README.md

```diff
@@ -887,10 +999,10 @@ failed, the names of those suites are emitted by `smartos-test` just before
 the script exits.
 
 When developers are adding tests to illumos, they should ensure that new
-tests are added to `$SRC/usr/src/pkg/manifests/*.mf` as these IPS
+tests are added to `$SRC/usr/src/pkg/manifests/\*.mf` as these IPS
```

Comment:
Markdown syntax: FWIW, that escaping shouldn't be necessary in backticks.

Reply:
vim disagrees, and it doesn't hurt rendering on github, so it's nicer to edit with... yeah, it's probably a vim bug.